doc/connectivity/region_extraction.rst

```diff
@@ -0,0 +1,186 @@
+.. _region_extraction:
+
+===================================================================
+Region Extraction for better brain parcellations
+===================================================================
+
+.. topic:: **Page summary**
+
+   This section shows how to use Region Extractor to extract each connected
+   brain regions/components into a separate brain activation regions and also
+   shows how to learn functional connectivity interactions between each
+   separate regions.
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+
+.. topic:: **References**
+
+   * `Abraham et al. "Region segmentation for sparse decompositions: better
+     brain parcellations from rest fMRI", Sparsity Techniques in Medical Imaging,
+     Sep 2014 <https://hal.inria.fr/hal-01093944>`_
+
+.. currentmodule:: nilearn.datasets
+
+Fetching resting state functional datasets
+==========================================
+
+We use ADHD resting state functional connectivity datasets of 20 subjects,
+which is already preprocessed and publicly available at
+`<http://fcon_1000.projects.nitrc.org/indi/adhd200/>`_. We use utilities
+:func:`fetch_adhd` implemented in nilearn for automatic fetching of these
+datasets.
+
+
+.. literalinclude:: ../../examples/connectivity/plot_extract_regions_canica_maps.py
+    :start-after: # utilities
+    :end-before: ################################################################################
```

Comment:
It's probably better to use a more unique marker to delimit the included snippet.